nimble/host/syscfg.yml

```diff
@@ -497,10 +507,21 @@ syscfg.defs:
         description: 'Minimum level for the BLE EATT log.'
         value: 1
 
+    BLE_HS_LEAUDIO_MOD:
```

Comment:
this LOGs could be separate patch. I think it was used in the Application using ISO right?